projects/packages/search/src/instant-search/class-instant-search.php

```diff
@@ -313,6 +313,15 @@ public function auto_config_search() {
 			return;
 		}
 
+		return $this->auto_config_search_no_pri();
+	}
+
+	/**
+	 * Automatically configure necessary settings for instant search
+	 *
+	 * @since  8.3.0
+	 */
+	public function auto_config_search_no_pri() {
```

Comment:
What does `no_pri` mean here?

Also, what do you think about overloading the original function call with a default parameter instead of creating a new function? Something like...

```php
	public function auto_config_search( $check_permissions = true ) {
		if ( $check_permissions && ! current_user_can( 'edit_theme_options' ) ) {
```

With this, we could invoke `Instant_Search::instance()->auto_config_search( false );` in the CLI.

Reply:
Good point. There was an ancient action called `wp_ajax_no_priv_your_action`, but still it's is not descriptive.
Had another consideration, I would rather just remove the privileges check here, b/c the function are called from APIs which have privilege checks of their own, and ARE not the same as the check in here.

REF1: https://github.com/Automattic/jetpack/blob/301c0f53f31adf0ea54cebd58bcbe67b387fe662/projects/plugins/jetpack/_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php#L1249
REF2: https://github.com/Automattic/jetpack/blob/301c0f53f31adf0ea54cebd58bcbe67b387fe662/projects/packages/search/src/class-rest-controller.php#L116   <- we probably want to widen the privilge check here

Reply:
Sounds good!